src/compiler/crystal/command.cr

```diff
@@ -462,6 +462,8 @@ class Crystal::Command
       error "You have input an invalid format, only text and JSON are supported"
     end
 
+    error "maximum number of threads cannot be lower than 1" if compiler.n_threads<1
```

Comment:
Please, fix the formatting: `compiler.n_threads < 1`

Reply:
Oh I forgot that. Fixed